.golangci.yml

```diff
@@ -29,7 +28,7 @@ linters-settings:
       See the License for the specific language governing permissions and
       limitations under the License.
 linters:
-  disable-all: false
```

Comment:
I think this would change the linters that run - is there a specific change you're looking to make here?
I can see the other changes are tidy up.